src/views/ContinuationOut.vue

```diff
@@ -681,18 +740,6 @@ export default class ContinuationOut extends Mixins(CommonMixin, DateMixin,
     }
 
     switch (this.staffPaymentData.option) {
```

Comment:
Did you leave this switch-case like this to be consistent with the other local filings? If not then I propose simplifying this.

You could also simplify the draft loading code (line 499).